apps/frontend/src/shared-research/api.ts

```diff
@@ -24,7 +26,35 @@ export const getResearchOutput = async (
   return resp.json();
 };
 
-export const getResearchOutputs = async (
+export const researchOutputFilters: Record<
+  ResearchOutputType,
+  { filter: string }
+> = {
+  Proposal: { filter: 'type:Proposal' },
+  Presentation: { filter: 'type:Presentation' },
+  Protocol: { filter: 'type:Protocol' },
+  Dataset: { filter: 'type:Dataset' },
+  Bioinformatics: { filter: 'type:Bioinformatics' },
+  'Lab Resource': { filter: 'type:Lab Resource' },
+  Article: { filter: 'type:Article' },
+};
+
+export const getResearchOutputs = (
+  { search }: SearchIndex,
+  options: GetListOptions,
+) =>
+  search<ResearchOutputResponse>(options.searchQuery, {
+    page: options.currentPage ?? 0,
+    hitsPerPage: options.pageSize ?? 10,
+    filters: [...options.filters]
+      .map(
+        (filter) => researchOutputFilters[filter as ResearchOutputType]?.filter,
```

Comment:
I want to avoid this cast ideally. I think i will reverse how this works matching known filters to whats in the URL